Dockerfile

```diff
@@ -9,13 +9,15 @@ RUN dnf install -y \
         fish \
         git \
         jq \
+        libffi-devel \
         moby-engine \
         pipx \
         podman \
         poetry \
         pre-commit \
         python \
         python-pip \
+        python3-devel \
         python3-libselinux \
         skopeo \
     && dnf group install -y \
```

Comment:
You can use `'@C Development Tools and Libraries'` to install it in the same command.